src/main/java/org/dominokit/domino/ui/forms/ValueBox.java

```diff
@@ -255,13 +255,15 @@ public T setRightAddon(Element rightAddon) {
     public T removeRightAddon() {
         if (nonNull(rightAddon)) {
             rightAddonContainer.removeChild(rightAddon);
+            rightAddon = null;
         }
         return (T) this;
     }
 
     public T removeLeftAddon() {
         if (nonNull(leftAddon)) {
             leftAddonContainer.removeChild(leftAddon);
```

Comment:
same as above

Reply:
Sure, whatever fixes the problem. You are the expert!